### PR TITLE
Add valid criteria prompt

### PR DESCRIPTION
**Pull request by WilliamRyank**
- Resize `Message Result Box` if its form type
- Add criteria of valid input for every attributes
- Allow user to type `/exit` to quit app when in form mode

### CODE REVIEW

File: src/main/java/organice/ui/FormUiManager.java [hunk 1]
```diff
@@ -38,24 +39,25 @@
  */
 public class FormUiManager {
 
-    private static final String COMMAND_EXIT = "/abort";
+    private static final String COMMAND_ABORT = "/abort";
+    private static final String COMMAND_EXIT = "/exit";
```
Comment (C-likethis123) [inline src/main/java/organice/ui/FormUiManager.java]: Wouldn't it be quite misleading to have an `/exit` command and another `exit` command? And why do we need to have different commands?

Comment (WilliamRyank) [reply src/main/java/organice/ui/FormUiManager.java]: cos during form maybe someones name is `exit` LOL

Comment (WilliamRyank) [reply src/main/java/organice/ui/FormUiManager.java]: Mr.exit

Comment (TheRealRavager) [reply src/main/java/organice/ui/FormUiManager.java]: Agree

**Pull request merged**